src/scss/components/_icons.scss

```
@@ -9,10 +9,23 @@
   font-display: swap;
 }
 
+.circle-icon {
+  background: $primary;
+  padding: 20px;
+  border-radius: 100%;
+}
+
+.label-icon {
+  color: $black !important;
+  vertical-align: bottom;
+}
```

```suggestion
.label-icon {
  vertical-align: bottom;
}
```

Utiliser classes utilitaires pour gérer les couleurs de texte : https://getbootstrap.com/docs/5.1/utilities/colors/

---

Du coup cette classe devient inutile, car on peut utiliser `align-bottom` à la place